tools/lib/provision_inner.py

```diff
@@ -253,8 +253,17 @@ def main(options: argparse.Namespace) -> int:
         dev_template_db_status = DEV_DATABASE.template_status()
         if options.is_force or dev_template_db_status == 'needs_rebuild':
             run(["tools/setup/postgres-init-dev-db"])
-            run(["tools/rebuild-dev-database"])
-            DEV_DATABASE.write_new_db_digest()
+            if options.skip_dev_db_build:
+                # We don't need to build the manual development
+                # database on CircleCI for running tests, so we can
+                # just leave it as a template db and save a minute.
+                #
+                # Important: We don't write a digest as that would
+                # incorrectly claim that we ran migrations.
+                pass
+            else:
+                run(["tools/rebuild-dev-database"])
```

Comment:
I removed the unnecessary `run(["tools/setup/postgres-init-dev-db"])` from here.